plugins/tech-insights-backend-module-jsonfc/src/service/JsonRulesEngineFactChecker.ts

```diff
@@ -141,7 +141,9 @@ export class JsonRulesEngineFactChecker
       );
     } catch (e) {
       if (isError(e)) {
-        throw new Error(`Failed to run rules engine, ${e.message}`);
+        throw new Error(`Failed to run rules engine, ${e.message}`, {
+          cause: e,
+        });
```

Comment:
`e` should just be the plain second argument directly, right? Not wrapped in an object. Or am I misunderstanding that?

Reply:
I am not sure what the right thing to do is. I had been following an example like this https://nodejs.org/api/errors.html#errorcause

Reply:
I am happy with either approach so long as I can retrieve the exception cause running the rules engine.

Reply:
Let's go with the one that's in the docs :shrug: